app/src/main/java/ch/epfl/favo/util/PictureUtil.java

```diff
@@ -37,6 +36,18 @@ public static PictureUtil getInstance() {
     return INSTANCE;
   }
 
+  public enum Folder {
+    FAVOR("favor/"),
+    CHAT("chat/"),
```

Comment:
I'm not sure if you used `PictureUtil` for uploading chat pictures to storage so I put these here in case. @danielpgp1012